docs/operations/distributed-tracing.md

```diff
@@ -0,0 +1,60 @@
+# Distributed Tracing
+
+## Modules
```

Comment:
We should update to use the same section headers, flow as the other sections. For example: https://github.com/nodeshift/nodejs-reference-architecture/blob/master/docs/functional-components/webframework.md

Reply:
Changes applied